### PR TITLE
Documented Ubuntu 22.04 LTS again

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Documented Ubuntu 22.04 LTS again (for 4.3.2.MU)

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [x] Manager-4.3

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
[suse_manager_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/9359047/suse_manager_client-configuration_guide.pdf)

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
Seems there's an error rendering one of the tables. Not sure if it affects master as well.

File: modules/client-configuration/pages/supported-features-ubuntu.adoc [hunk 1]
```diff
@@ -38,211 +35,253 @@ ifeval::[{suma-content} == true]
 | {ubuntu}{nbsp}16.04
 | {ubuntu}{nbsp}18.04
 | {ubuntu}{nbsp}20.04
+| {ubuntu}{nbsp}22.04
```
[the remark this answers is not in the document]
Comment (keichwa) [reply modules/client-configuration/pages/supported-features-ubuntu.adoc:38]: thanks for spotting.  master is probably also effected.
[suse_manager_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/9371933/suse_manager_client-configuration_guide.pdf)

**Review by jcayouette — APPROVED** [review]

**Comment by jcayouette** [comment]
I approved but what about this rendering of the pdf table?

**Comment by keichwa** [comment]
> I approved but what about this rendering of the pdf table?

According to my build, the table is now fixed. Thanks for checking!

**Pull request merged**